lalrpop/Cargo.toml

```
@@ -26,7 +26,7 @@ ena = { version = "0.14", default_features = false }
 is-terminal = "0.4.2"
 itertools = { version = "0.10", default_features = false, features = ["use_std"] }
 petgraph = { version = "0.6", default_features = false }
-regex = { workspace = true }
+regex = { workspace = true, features = ["std"] }
```

```suggestion
regex = { workspace = true }
```
Is this redundant because we already have it as workspace dependency?